backend/core/utils/constants.py

```diff
@@ -2,35 +2,6 @@
 Define constants for Tournesol's main configuration
 """
 
-from settings.settings import CRITERIAS_DICT, CRITERIAS, DRF_RECAPTCHA_PUBLIC_KEY
-
-comments = {
-    "featureNames": "Dictionary mapping quality features to their description",
-    "featureList": "List of quality features in the correct order",
-    "featureColors": "Colors for quality features",
-    "SEARCH_DIVIDER_COEFF": "divide search score by this value",
-    "SEARCH_FEATURE_CONST_ADD": "add this value to Tournesol scores",
-    "DRF_RECAPTCHA_PUBLIC_KEY": "Public ReCaptcha key",
-    "featureIsEnabledByDeFault": "Is the feature enabled by-default?",
-    "YOUTUBE_VIDEO_ID_REGEX_SYMBOL": "Regular expression of YouTube videos (one symbol)",
-    "MIN_NUM_RATE_LATER": "Minimal number of videos to rate before redirecting to rating page",
-    "DEFAULT_PREFS_VAL": "Default value of preferences for tournesol_score",
-    "N_PUBLIC_CONTRIBUTORS_SHOW": "Number of public contributors to show",
-}
-
-featureColors = {
-    "largely_recommended": "#ff00ff",
-    "reliability": "#6573c3",
-    "pedagogy": "#8561c5",
-    "importance": "#ff784e",
-    "engaging": "#ffc107",
-    "backfire_risk": "#e91e63",
-    "layman_friendly": "#50f22c",
-    "diversity_inclusion": "#2ceff2",
-    "better_habits": "#c6eb34",
-    "entertaining_relaxing": "#6ba4ff",
-}
```

Comment:
Unrelated to the PR, quick thought: 

If we want multiple frontends to share the same constants, would a good manner to achieve this be to serve the constants from the backend API? 

It may be a good design for Tournesol-As-A-Service, if we want different frontends from the same code being configured differently simply by connecting to a different backend.

Reply:
Yes, fetching such context from the API would make sense.   
It could probably include more than just constants though: names/icons of the criteria, whether they are optional or not, available filters, etc. Why not even configurable via some kind of back-office. Some discussion would be needed to find the right level of abstraction here.